Gulpfile.mjs

```diff
@@ -388,19 +420,23 @@ function buildRollup(packages, targetBrowsers) {
             }),
             rollupNodeResolve({
               extensions: [".ts", ".js", ".mjs", ".cjs", ".json"],
-              browser: targetBrowsers,
-              exportConditions: targetBrowsers ? ["browser"] : [],
+              browser: buildStandalone,
+              exportConditions: buildStandalone ? ["browser"] : [],
               // It needs to be set to 'false' when using rollupNodePolyfills
               // https://github.com/rollup/plugins/issues/772
-              preferBuiltins: !targetBrowsers,
+              preferBuiltins: !buildStandalone,
             }),
             rollupJson(),
-            targetBrowsers &&
+            buildStandalone &&
               rollupPolyfillNode({
                 sourceMap: sourcemap,
                 include: "**/*.{js,cjs,ts}",
```

Comment:
Not related to this PR: will there be ".mjs"?

Reply:
We are already handling `mjs` in `rollupBabelPlugin`. I guess we should be consistent here.